remo/base/static/base/css/app-fd4.less

```diff
@@ -2135,32 +2134,42 @@ meter::-o-meter-suboptimum-value {
         .copyblock {
             padding-bottom: 10px;
         }
-        .required-field {
-            top: -30px;
-            margin-bottom: -30px;
-        }
         button {
             margin-top: 10px;
         }
     }
     .category-select {
         margin-top: 10px;
-        .required-field {
-            top: 10px;
-        }
-    }
-    .map-point-select {
-        button + .required-field {
-            top: 10px;
-        }
     }
+    
     #event-attend-button button {
         padding: 8px 10px 10px;
     }
     .event-single-title {
         font-weight: bold;
         margin-bottom: 30px;
     }
+
+    .section {
+        padding-right: 25px;
+        padding-bottom: 25px;
+    }
+
+    .required-field {
+        &:extend(.required-field);
+        left: 5px;
+        top:0;
```

Comment:
Minor tweak: try this to be `top: 10px`, so it's vertical aligned to the middle of the input field.